doc/Style.md

```diff
@@ -350,7 +350,7 @@ import agora.test.Base; // Base functions for the test environment
 unittest
 {
     const NodeCount = 4;
```

Comment:
makeTestNetwork can no longer figure out the default type after signature change(seems Dlang compiler problem)

Reply:
Having to modify all tests is a non-starter. I'll take a look at the failure soon. The rest looks good.